### PR TITLE
Components Added

### PR DESCRIPTION
**Pull request by prathmeshkhandelwal1**
savory-bears.surge.sh

### CODE REVIEW

**Comment by prathmeshkhandelwal1** [comment]
New surge link is cerebro60.surge.sh

**Review by thepushkarp — CHANGES_REQUESTED** [review]
Add media queries for widths of  1080 and 400

**Comment by thepushkarp** [comment]
Any further updates on this @prathmeshkhandelwal1 ?

**Comment by theyashshahs** [comment]
@prathmeshkhandelwal1 resolve the merge conflicts as soon as possible.

**Review by samagragupta — COMMENTED** [review]
Updated surge link?

File: src/Components/Home/Cover/Cover.scss [hunk 1]
```diff
@@ -74,8 +74,8 @@
   }
 
   &__image {
-    width: 400px;
-    height: 400px;
+    width: 26vw;
```
Comment (rossoskull) [inline src/Components/Home/Cover/Cover.scss]: Don't use `vw`, we want fixed size of image.

File: src/Components/Home/Cover/Cover.scss [hunk 2]
```diff
@@ -200,3 +200,120 @@
     }
   }
 }
+.Recpurple1{
+  width: 3.3vw;
```
Comment (rossoskull) [inline src/Components/Home/Cover/Cover.scss]: Same here.

File: src/Components/Home/Cover/Cover.scss [hunk 3]
```diff
@@ -200,3 +200,120 @@
     }
   }
 }
+.Recpurple1{
+  width: 3.3vw;
+  height: 3.3vw;
+  position: relative;
+  top: -3.5vw;
```
Comment (rossoskull) [inline src/Components/Home/Cover/Cover.scss]: Use `%` or `px` for positioning

**Review by rossoskull — CHANGES_REQUESTED** [review]

**Review by rossoskull — APPROVED** [review]

**Pull request merged**